script.filecleaner/utils.py

```diff
@@ -0,0 +1,275 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+import os
+import re
+import time
+from ctypes import *
+
+import xbmcgui
+import xbmcvfs
+from settings import *
+
+
+# Addon info
+__addonID__ = "script.filecleaner"
+__addon__ = Addon(__addonID__)
+__title__ = __addon__.getAddonInfo("name")
+__profile__ = xbmc.translatePath(__addon__.getAddonInfo("profile")).decode("utf-8")
+__icon__ = xbmc.translatePath(__addon__.getAddonInfo("icon")).decode("utf-8")
+
```

Comment:
We are trying to move away from using \__name\__ type labels. They are reserved for python internals. Please clean them up in your next update.